guide/additional-features/reloading-commands.md

```diff
@@ -0,0 +1,65 @@
+# Reloading Commands
+
+When writing your commands, you may find it tedious to restart your bot every time for testing the smallest changes. With a command handler, you can eliminate this issue and reload your commands while your bot is running.
+
+::: warning
+ESM does not support require and clearing import cache. You can use [hot-esm](https://www.npmjs.com/package/hot-esm) to import files without cache.
+:::
```

Comment:
It appears that Windows support is experimental (https://github.com/vinsonchuong/hot-esm/issues/33). I think it would be a good idea to state this whilst this is an open issue.